modules/effects/src/main/scala/dev/profunktor/redis4cats/redis.scala

```diff
@@ -426,7 +445,7 @@ object Redis {
 
   }
 
-  def apply[F[_]: MkRedis: MonadThrow]: RedisPartiallyApplied[F] = new RedisPartiallyApplied[F]
+  def apply[F[_]: MkRedis: Sync]: RedisPartiallyApplied[F] = new RedisPartiallyApplied[F]
```

Comment:
The idea behind `MkRedis` is to avoid any need for `Sync / Async`. This could be fixed by moving the following method to the `MkRedis` instance:

```scala
Sync[F].blocking(Runtime.getRuntime.availableProcessors())
```

Reply:
@gvolpe  fixed in 6d1d873

Reply:
@gvolpe  I'm not sure if adding to MkRedis a private function is actually backwards compatible, if someone has like a test instance or something they are using in code it could break for them.